### PR TITLE
Audit view

### PR DESCRIPTION
**Pull request by tmb**

### CODE REVIEW

**Comment by maxwofford** [comment]
@tmb: Removed myself from review until the features we talked about are implemented. Once they are, tag me

File: app/controllers/static_pages_controller.rb [hunk 1]
```diff
@@ -27,6 +27,33 @@ def index
     end
   end
 
+  def audit
+    return false unless admin_signed_in
```
Comment (lachlanjc) [inline app/controllers/static_pages_controller.rb]: Any reason to not make this a policy?

Comment (tmb) [reply app/controllers/static_pages_controller.rb]: yeah, pundit isn't being used in this controller anywhere

Comment (maxwofford) [reply app/controllers/static_pages_controller.rb]: Why not add it?

Comment (tmb) [reply app/controllers/static_pages_controller.rb]: good point, will do

**Comment by thesephist** [comment]
@tmb is this something we want to push out?

**Pull request closed without merging**